Haneke/Haneke.swift

```diff
@@ -49,14 +49,14 @@ public struct Shared {
     }
 }
 
-func errorWithCode(code : Int, #description : String) -> NSError {
+public func errorWithCode(code : Int, #description : String) -> NSError {
```

Comment:
This should not be public. I don't want API clients to generate Haneke-labeled errors.